addon/index.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
The purpose of the `index.js` file in general? It allows someone to do something like:

```js

import {Button} from 'ember-frost-core'
```

instead of 
```js
import Button from 'ember-frost-core/components/frost-button'
```
By allowing the former, you don't consider the path to a component within your addon as part of your public API. If you export the thing at the root of the addon, then you can move it around within your addon (maybe someone decides pods are the way to go, or something like that) you don't impact your consumers at all. 

We do this in bunsen as well: https://github.com/ciena-frost/ember-frost-bunsen/blob/master/addon/index.js